doc/source/parameters/cfd/dynamic_flow_control.rst

```diff
@@ -0,0 +1,38 @@
+Dynamic flow control
+~~~~~~~~~~~~~~~~~~~~
+
+This subsection's purpose is to enable a dynamic flow control. It enables setting a volumetric flow on a specific boundary toward to normal of the wall. 
+It calculates a Beta coefficient at each time step and may be added to a source term. If the chosen wall boundary on which a volumetric flow is imposed is at inlet, 
+the volumetric flow rate targeted must be negative since outward normal vector is used. Thereby, if the chosen wall boundary is at outlet, 
+volumetric flow rate must be positive.
+
+The default parameters are:
+
+.. code-block:: text
+
+  subsection flow control
+    set enable = false
+    set volumetric flow rate = 0
+    set boundary id = 0
+    set flow direction = 0
+    set initial beta = 0
+  end
+
+* The ``enable`` parameter is set to ``true`` to enable an imposed volumetric flow rate.
+
+* The ``volumetric flow rate`` parameter specifies the flow rate (:math:`m^{dim}/s`). 
+
+.. warning::
+
+  This value should be negative if imposed on an inlet boundary, and positive if imposed on an outlet boundary.
+
+.. note::
+
+  When there is a periodic boundary, the wall boundary should be at inlet only and if the chosen wall boundary is at inlet, flow rate must be negative and vice versa.
```

Comment:
I don't really understand this sentence. Does it mean that you should only control the flow at the inlet if there are periodic boundary conditions in the problem? Is it possible to control the flow at several boundary ids at the same time?

Reply:
This is what I understood from the previous documentation. I think you can only control a volumetric flow on one boundary

Reply:
Ah ok! So maybe we can rephrase it as this: 

If you are simulating a problem with periodic boundary conditions at inlet and outlet walls, it is important to control the volumetric flow rate specifying the boundary id that corresponds to the inlet. 

What do you think?

Reply:
that looks great!